include/ttmlir/Utils.h

```diff
@@ -183,6 +183,21 @@ inversePermutation(llvm::ArrayRef<int64_t> permutation) {
   return inversePermutation;
 }
 
+// Returns a vector `mergedPermutation`, such that
+// mergedPermutation[i] = permutation1[permutation2[i]], for all i. Assumes
+// that permutation1 and permutation2 have the same size. Example:
+// permutation1 = [1, 2, 0], permutation2 = [2, 0, 1] -> [0, 1, 2]
+inline llvm::SmallVector<int64_t>
+mergePermutations(llvm::ArrayRef<int64_t> permutation1,
```

Comment:
Hmmm yea I missed it because applyPermutation didn't sound like thing I needed. Maybe we leave mergePermutations function and instead of logic I added call apply? I would be easier for people to find it when they need it and it's more clear when reading the code what is the intent?